### PR TITLE
It's my projects

### PR DESCRIPTION
**Pull request by VeryHappy2**
My ptojects

### CODE REVIEW

File: My project2/Calculator/Program.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
+﻿using System;
+
+while (true)
+{
+    Console.WriteLine("Please write first number, + or / or * and second number");
+
+    if (!double.TryParse(Console.ReadLine(), out double number1) ||
+    !char.TryParse(Console.ReadLine(), out char a) ||
```
Comment (petrochernykh) [inline My project2/Calculator/Program.cs]: 'a' is a bad name for variable and why you need 3 TryParse() here?

File: My project2/Calculator/Program.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    !double.TryParse(Console.ReadLine(), out  double number2))
+    {
+        Console.WriteLine("Please write valid numbers.");
+        break;
```
Comment (petrochernykh) [inline My project2/Calculator/Program.cs]: I think we don't need **break** here

File: My project2/My project/Program.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    Console.WriteLine("Do you want see note?");
+    string user = Console.ReadLine();
+
+    if (user.ToLower() == "no" )
+    {
+        Console.Clear(); 
+    }
+    else if (user.ToLower() == "yes" || user.ToLower() == "yeah") 
+    {
+
```
Comment (petrochernykh) [inline My project2/My project/Program.cs]: We musn't leave empty else if statement, as in this case we don't need else_if at all and we can remove this

File: My project2/My project/Program.cs [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    {
+        
+    }
+}
+
+
+while (a == false)
+{
+    Note();
+}
```
Comment (petrochernykh) [inline My project2/My project/Program.cs]: For what we need this part?

File: Крестики нолики/Program.cs [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+            win = -1;
+            Console.WriteLine("Draw!");
+
+        }
+        else if (player != 8)
+        {
+
+            if (player % 2 == 0)
+            {
+                choice = int.Parse(Console.ReadLine());
```
Comment (petrochernykh) [inline Крестики нолики/Program.cs]: It's better to use .TryParse()

File: Крестики нолики/Program.cs [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+    {
+        Console.WriteLine("Write please the numbers that are on the board");
+    }
+
+
+} while (win != 1 && win != -1);
+if (win == 1) 
+{
+    Console.WriteLine($" Player: {player % 2 + 1} win! ");
+}
```
Comment (petrochernykh) [inline Крестики нолики/Program.cs]: That's great that you decided to write this game. It's interesting and will help you to improve yourself.

**Review by VeryHappy2 — COMMENTED** [review]
It's "MathGame"

File: My project2/MathGame/Program.cs [hunk 7]
```diff
@@ -0,0 +1,69 @@
+﻿Random random = new Random();
+try
+{
+    Console.WriteLine("Please write.Until what will be numbers the first and second?");
+
+    int firstnumber = Convert.ToInt32(Console.ReadLine()); // first number random
```
Comment (petrochernykh) [inline My project2/MathGame/Program.cs]: What if user decided to send you character instead of number? It's better to your TryParse()

**Pull request merged**